lib/genome/normalizers/interaction_claim_type.rb

```diff
@@ -42,12 +61,34 @@ def self.add_unless_exists(type, interaction_claim)
         end
       end
 
+      def self.add_unless_exists_for_interaction(type, interaction)
+        unless interaction.interaction_types.include?(type)
+          interaction.interaction_types << type
+        end
+      end
+
       def self.name_normalizer(val)
         val = val.downcase
         if val == 'na' || val == 'n/a'
           'n/a'
         elsif val =~ /other/ || val =~ /unknown/
           'other/unknown'
+        elsif val == 'neutralizer' || val == 'reducer' || val == 'metabolizer' || val == 'acetylation' || val == 'chelator' || val == 'cross-linking/alkylation' || val == 'positive modulator'
```

Comment:
@kcotto we are normalizing many concepts to `negative modulator`, but here we're normalizing `positive modulator` to `modulator`. Why are we stepping back from using `positive modulator` as a normalized concept?

Reply:
The only concept of a postitive modulator we currently have in our table is `positive allosteric modulator`. I just wasn't sure if we wanted to add `positive modulator` to the table as well. If we do, I can totally make that change.

Reply:
I think that we should keep to the notion of a `positive modulator` where we can, though it is appropriate to normalize to an unspecified modulator type if we think it is neutral, or do not have enough information to normalize to negative / positive. In general, I think that keeping to the positive/neutral/negative definitions [as described on Wikipedia](https://en.wikipedia.org/wiki/Allosteric_modulator) is safe.

Reply:
So `positive modulator` should be kept as such. Should `positive allosteric modulator` then also be normalized to `positive modulator`? I don't see it in the list currently.

Reply:
Yeah, I think the plan was to normalize `positive allosteric modulator` to `positive modulator` to keep consistent with also normalizing concepts to `negative modulator`

Reply:
This change has been implemented.